### ISSUE TITLE
Improve aria-live region updates

### ISSUE THREAD

**Issue by btzr-io** [btzr-io/godot-aria#1]
Currently the plugin enforces aria-live assertive, it should have a way to toggle this:

> Because an interruption may disorient users or cause them to not complete their current task, don't use the assertive value unless the interruption is imperative.

See [Using aria-live](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-live#using_aria-live)

Also it should support additional live region [attributes](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions#additional_live_region_attributes)
